src/Core/Impl/Microsoft.Python.Core.csproj

```diff
@@ -23,6 +23,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="2.2.0" />
```

Comment:
Is this added only for the matcher?

Reply:
Yes. It would be great to avoid it

Reply:
`DirectoryInfo,EnumerateFileSystemInfos()`?

Reply:
Yes, I suggested adding similar method with exclusion and inclusion options.